docs/sync-rules/GMAIL.md

```diff
@@ -0,0 +1,56 @@
+### Setting up the GMail connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+### Example advanced sync rules
+
+See [search operators you can use with Gmail](https://support.google.com/mail/answer/7190) for all supported operators.
+
+#### Emails from the year 2022 or later
+
+```json
+[
+  {
+    "messages": [
+      "after:2021/12/31"
+    ]
+  }
+]
```

Comment:
Shouldn't it be an object instead of an array?
https://github.com/elastic/connectors-python/blob/302d74389c9c114a6c466a19aa1ea2ba76b911de/connectors/sources/gmail.py#L311

```suggestion
{
  "messages": [
    "after:2021/12/31"
  ]
}
```

Reply:
Yes, you're right! Good catch, will adjust it